### DISCUSSION TITLE
Homebrew Uninstalled Ventura 13.5 2021 M1 MacBook Pro

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#4694] [Getting started] [answered]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

Hi there. I've used Homebrew on my 2021 MacBook Pro laptop (I use zsh, iTerm 2 and oh-my-zsh to prettify my terminal) for months with absolutely zero problems. Last night I was using Homebrew, and everything was working fine. Yet sometime between last night and this afternoon when I went to use Homebrew again, it uninstalled from my computer. I ran a disk check to see if there was a problem with my machine's file system, but my computer was fine. I ran some malware scans and everything came back clean. I checked my path and the path was fine. It's like my Homebrew installation just vanished. If anyone has experienced similar problems or has any suggestions for how to prevent this from happening in the future, I would greatly appreciate it.

**Comment by osalbahr** [comment]
What does `file /opt/homebrew/bin/brew /usr/local/bin/brew` tell you?

**Reply by ghost** [reply]
This was the output:
`/opt/homebrew/bin/brew: Bourne-Again shell script text executable, ASCII text
/usr/local/bin/brew:    cannot open '/usr/local/bin/brew' (No such file or directory)`

**Reply by ghost** [reply]
This is the output of `echo $PATH`: `/Users/username/.npm-global/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin`

**Reply by ghost** [reply]
The command `which Homebrew` returns `Homebrew not found`. `which brew` returns `brew not found`. I've also tried checking in my `/usr/local/bin` for Homebrew but can't find anything.

**Reply by ZhongRuoyu** [reply, marked as answer]
Please do the following, as suggested in https://github.com/orgs/Homebrew/discussions/775#discussioncomment-359960:

```
==> Next steps:
- Add Homebrew to your PATH in /Users/homefolder/.zprofile:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
- Run `brew help` to get started
```

(Substitute `homefolder` with your username.)

**Reply by ghost** [reply]
Thank you so much! This fixed the issue. I had seen this thread but wasn't sure if it applied in my case, since Homebrew has been installed and working fine for a while...did this problem result from a change in my path?

**Reply by ZhongRuoyu** [reply]
I'm not exactly sure why, but your `~/.zprofile` may have been changed somehow.

**Reply by ghost** [reply]
I just checked and you were right, my `~/.zprofile` was changed. Thanks so much for all your help!
